genesyscloud/resource_genesyscloud_routing_skill_group.go

```diff
@@ -309,6 +427,58 @@ func readSkillGroups(ctx context.Context, d *schema.ResourceData, meta interface
 			d.Set("skill_conditions", nil)
 		}
 
+		log.Printf("Reading skills group %s member divisions", d.Id())
```

Comment:
Again this function is getting big.  The 'readMemberDivision' logic is cohesive and should be extracted into its own functions